mmv1/templates/terraform/examples/vertex_ai_model_source_basic.tf.tmpl

```diff
@@ -0,0 +1,7 @@
+resource "google_vertex_ai_models" "{{$.PrimaryResourceId}}" {
+  display_name = "{{index $.Vars "display_name"}}"
+  project = "{{index $.Vars "project_name"}}"
+  source_model = "projects/{{index $.Vars "project_name"}}>/locations/us-central1/models/4469245519628878651"
```

Comment:
Is this model only present in our HashiCorp-owned testing project? If so, this test will eventually fail when running in automation on this PR as the source model will be inaccessible.

Once we get the test passing using the HashiCorp testing project we could make the acceptance test handwritten and try to navigate ensuring how the project used for testing has a model present at the start of the test.